glean-core/src/metrics/mod.rs

```diff
@@ -30,27 +34,44 @@ pub use self::string_list::StringListMetric;
 pub use self::time_unit::TimeUnit;
 pub use self::uuid::UuidMetric;
 
+/// The available metrics
+///
+/// This is the in-memory and persisted layout of a metric
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub enum Metric {
+    /// A boolean metric. See [`BooleanMetric`](struct.BooleanMetric.html) for more information.
```

Comment:
Ah, doesn't autolink/discover things?

Reply:
So there is an [intra-doc links RFC](https://github.com/rust-lang/rfcs/blob/master/text/1946-intra-rustdoc-links.md) and it seems part of this is implemented, but I don't think it's stable yet (at least I didn't get it to work, I'll ask around)

Reply:
Ah, interesting. I'm fine with leaving this as it is for now, as long as we file a bug to fix it in the future :)